src/XMakeBuildEngine/UnitTests/BackEnd/TargetUpToDateChecker_Tests.cs

```diff
@@ -903,6 +909,122 @@ bool includeOutput2
                 if (File.Exists(output2)) File.Delete(output2);
             }
         }
+
+        private static readonly DateTime Old = new DateTime(2000, 1, 1);
+        private static readonly DateTime Middle = new DateTime(2001, 1, 1);
+        private static readonly DateTime New = new DateTime(2002, 1, 1);
+
+        [Fact(Skip = "Creating a symlink on Windows requires elevation.")]
```

Comment:
Yes, this is awful. `xunit.netcore.extensions` has a `ConditionalFact` that will allow for this:
```C#
[ConditionalFact(nameof(RunningWithElevatedPrivileges))]
```
(from [here](https://github.com/dotnet/corefx/blob/c2a6a128a6d2cdbad3a8be663d027152ca9d63bc/src/System.ServiceProcess.ServiceController/tests/System.ServiceProcess.ServiceController.Tests/ServiceControllerTests.cs#L97).)

But we're not using that in master yet and my quick effort to drag it in failed, so I opted to just skip the tests for now.